Mixpanel/Mixpanel.m

```diff
@@ -696,6 +722,7 @@ - (void)archiveProperties
     [p setValue:self.people.unidentifiedQueue forKey:@"peopleUnidentifiedQueue"];
     [p setValue:self.shownNotifications forKey:@"shownNotifications"];
     [p setValue:self.timedEvents forKey:@"timedEvents"];
+    [p setValue:self.automaticEventsEnabled forKey:@"automaticEvents"];
```

Comment:
since a user can disable pre-track at any point i think we should only rely on `decide` responses and not store this information in disk. Otherwise, we'll send pre-track events even after the customer disabled this option